ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -27,12 +27,12 @@ open class TableViewHeaderFooterView: UITableViewHeaderFooterView {
         case regular
         case primary
 
-        func textColor(for window: UIWindow) -> UIColor {
+        func textColor(fluentTheme: FluentTheme) -> UIColor {
             switch self {
             case .regular:
-                return Colors.Table.HeaderFooter.accessoryButtonText
+                return UIColor(dynamicColor: DynamicColor(light: fluentTheme.globalTokens.sharedColors[.red][.shade10], dark: fluentTheme.globalTokens.sharedColors[.red][.tint30]))
```

Comment:
This doesn't seem right--I don't believe our default accessory text color should be red?

Reply:
Thanks for catching this.